README.md

```diff
@@ -1 +1,138 @@
-# proxy
\ No newline at end of file
+# Open Runtimes Proxy 🚦
+
+![open-runtimes-box-bg-cover](https://user-images.githubusercontent.com/1297371/151676246-0e18f694-dfd7-4bab-b64b-f590fec76ef1.png)
+
+---
+
+[![Discord](https://img.shields.io/discord/937092945713172480?label=discord&style=flat-square)](https://discord.gg/mkZcevnxuf)
+[![Build Status](https://github.com/utopia-php/balancing/actions/workflows/tester.yml/badge.svg)](https://github.com/utopia-php/balancing/actions/workflows/tester.yml)
+[![Twitter Account](https://img.shields.io/twitter/follow/appwrite?color=00acee&label=twitter&style=flat-square)](https://twitter.com/appwrite)
+
+<!-- [![Docker Pulls](https://img.shields.io/docker/pulls/appwrite/appwrite?color=f02e65&style=flat-square)](https://hub.docker.com/r/appwrite/appwrite) -->
```

Comment:
You can use the link to the proxy docker hub image

Reply:
It will only work after we release first version. Added it.